src/apps/containers.clj

```diff
@@ -18,7 +19,8 @@
         [korma.core :exclude [update]]
         [korma.db :only [transaction]])
   (:require [clojure.tools.logging :as log]
-            [korma.core :as sql]))
+            [korma.core :as sql]
+            [clojure.string :as string]))
```

Comment:
I've been trying to keep `:require` lists in alphabetical order, especially once they get long.